3p/iframe-messaging-client.js

```diff
@@ -59,7 +59,7 @@ export class IframeMessagingClient {
   /**
    *  Send a postMessage to Host Window
    *  @param {object} message The message to send.
-   *  @private
+   *  @protected
```

Comment:
Why `@protected`? Why should this be overridden? Trailing underscore means `@private`, so would have to be removed.